.prbuilds/setup.playbook.yml

```diff
@@ -4,26 +4,26 @@
     AWS_PROFILE: frontend
     ANSIBLE_HOST_KEY_CHECKING: False
   tasks:
-        
+
     - name: preinstall nvm
       shell: curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
-    
+
     - name: nvm install correct version
       shell: . ~/.nvm/nvm.sh; nvm install
       args:
         chdir: ~/workspace/dotcom-rendering
         executable: /bin/bash
-    
+
     - name: run make dev
-      command: screen -S trousers-build -d -m make dev
+      command: screen -S trousers-build -d -m bash -c '. ~/.nvm/nvm.sh; nvm use; make dev'
```

Comment:
This is running PRBuilds against the dev server, which is not optimised for performance. If PR Builds is running Lighthouse against the dev server, it will give lower scores than we would see in production. 

Would it be better / possible to run `make build start` and test against the PROD server on port 9000?

Reply:
> Would it be better / possible to run make build start and test against the PROD server on port 9000?

The main problem with that is: prod server needs to get the datamodel pushed to dotcom-rendering on the request. I don't think the ?url param works in prod for dotcomponents? I might be wrong.

If we can have a way to make dotcomponents get data itself from frontend despite running in production mode, we can use that.